retired/shiro/security/shiro/pom.xml

```
@@ -80,6 +80,8 @@
         <dependency>
             <groupId>org.apache.shiro</groupId>
             <artifactId>shiro-core</artifactId>
+            <classifier>jakarta</classifier>
+            <version>2.0.2</version>
```

You should also update the `shiro.version` in `bom/pom.xml`.

```suggestion
            <version>${shiro.version}</version>
```